gitlab_issues.inc.php

```diff
@@ -20,14 +20,14 @@ function plugin_gitlab_issues_convert() {
 import dayjs from 'https://cdn.pika.dev/dayjs@1.11.6';
 
 document.addEventListener('DOMContentLoaded', async () => {
-  const issues = (await getIssues()).slice(0, $limit);
+  const issues = (await getIssues('$domain', '$project_id')).slice(0, $limit);
   const issuesTableDOM = makeIssuesTableDOM(issues);
-  document.querySelector('#gitlab-issues-placeholder').appendChild(domify(issuesTableDOM));
-  document.querySelector('#gitlab-issues-loading').style = 'display: none;';
+  document.querySelector('#gitlab-issues-placeholder-$project_id').appendChild(domify(issuesTableDOM));
+  document.querySelector('#gitlab-issues-loading-$project_id').style = 'display: none;';
 });
 
-async function getIssues() {
-  const url = 'https://$domain/api/v4/projects/$project_id/issues?order_by=updated_at';
+async function getIssues(domain, project_id) {
+  const url = 'https://' + domain + '/api/v4/projects/' + project_id + '/issues?order_by=updated_at';
```

Comment:
```suggestion
  const url = `https://${domain}/api/v4/projects/${project_id}/issues?order_by=updated_at`;
```

PHPと混ざっていてわかりにくいですが、ここはJavaScriptなのでtemplate literalを使うともう少しきれいに書けそうです。

Reply:
jsでこの記法が使えるの知らなかったです。ありがとうございます！

Reply:
もしかして`$`を使うとPHPに解釈されておかしくなってしまうんでしょうか？ その場合このままで問題ないですね。

Reply:
おそらくエスケープが必要ですね

Reply:
修正します

Reply:
すみません、ありがとうございます。それ以外は特に問題なさそうです。